logger/log.go

```diff
@@ -0,0 +1,65 @@
+package logger
+
+import (
+	"fmt"
+	"log"
+)
+
+const (
+	// ANSI color codes
+	red    = "\033[31m"
+	green  = "\033[32m"
+	yellow = "\033[33m"
+	white  = "\033[97m"
+	reset  = "\033[0m"
+)
+
+func Successf(format string, args ...interface{}) {
+	msg := green + fmt.Sprintf(format, args...) + reset
+	log.Printf(msg)
+}
+
+func Successln(args ...interface{}) {
+	msg := green + fmt.Sprint(args...) + reset
+	log.Println(msg)
+}
```

Comment:
How about reusing more of the code to reduce replication:
```suggestion
func Successln(args ...interface{}) {
	Successf(fmt.Sprint(args...))
}
```
or a slightly more efficient variant:
```suggestion
func Successln(args ...interface{}) {
	Successf("%s", args...)
}
```